File: JavaScript/Faccat/Exercicio83.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let A = new Array(20);

rl.prompt();

rl.on('line', (input) => {
  for (let indice = 0; indice < 20; indice++) {
    A[indice] = parseFloat(input);
    rl.prompt();
  }
}).on('close', () => {
  console.log("Elementos do vetor na ordem inversa:");
  for (let indice = 19; indice >= 0; indice--) {
    console.log(A[indice]);
  }
  process.exit(0);
});

File: Java/src/Faccat/Exercicio83.java
package Faccat;

import java.util.Scanner;

public class Exercicio83 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double[] A = new double[20];

        for (int indice = 0; indice < 20; indice++) {
            System.out.println("Digite um número:");
            A[indice] = scanner.nextDouble();
        }

        System.out.println("Elementos do vetor na ordem inversa:");
        for (int indice = 19; indice >= 0; indice--) {
            System.out.println(A[indice]);
        }

        scanner.close();
    }
}

